### PR TITLE
:sparkles: selectbox

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -118,12 +118,14 @@
...
     "classnames": "^2.2.5",
     "react": "^15.4.2",
+    "react-select": "^1.0.0-rc.3",
     "recompose": "^0.22.0"
   },
   "dependencies": {
     "react-img-fallback": "^1.0.0",
     "react-ink": "^6.1.1",
     "react-redux": "^5.0.3",
+    "react-select": "^1.0.0-rc.3",
```
Comment (fabienjuif) [inline package.json]: move it to `devDependencies`

File: src/selectBox/selectBox.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const classes = classnames(
+    styles.select,
+    className,
+  )
+
+  return (
+    <div className={classes} style={style}>
+      {label && <label htmlFor={name}>{label}{required && '*'}</label>}
+      <Select
+        noResultsText="Aucun résultat"
```
Comment (fabienjuif) [inline src/selectBox/selectBox.jsx]: french

File: src/selectBox/selectBox.styles.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+@import '~styles/theme';
+
+.select {
+  :global(.Select .Select-control) {
+    background-color: $default-bg-color;
+
+    :global(.Select-value) {
+      max-width: 90%;
```
Comment (fabienjuif) [inline src/selectBox/selectBox.styles.scss]: Why no em ?

File: src/selectBox/selectBox.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  required: PropTypes.bool,
+}
+
+SelectBox.defaultProps = {
+  label: undefined,
+  value: undefined,
+  required: false,
+  className: '',
+  style: {},
+  options: [],
```
Comment (fabienjuif) [inline src/selectBox/selectBox.jsx]: options is not listed in propTypes

File: src/selectBox/selectbox.spec.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <Provider store={store}>
+      <Decorated {...rest} name={name} />
+    </Provider>,
+  )
+
+  const tree = component.toJSON()
+  expect(tree).toMatchSnapshot()
+}
+
+describe('common/SelectBox', () => {
```
Comment (fabienjuif) [inline src/selectBox/selectbox.spec.js]: no `value` test

File: src/selectBox/selectBox.styles.scss (hunk 3 above)
Comment (guillaumecrespel) [inline src/selectBox/selectBox.styles.scss]: easy to calcul

**Pull request merged**